man/org.freedesktop.systemd1.xml

```diff
@@ -12445,7 +12472,8 @@ $ gdbus introspect --system --dest org.freedesktop.systemd1 \
       <varname>ManagedOOMMemoryPressureDurationUSec</varname>,
       <varname>ProtectControlGroupsEx</varname>, and
       <varname>PrivatePIDs</varname> were added in version 257.</para>
-      <para><varname>ProtectHostnameEx</varname> was added in version 258.</para>
```

Comment:
Does it still makes sense to also have ProtectHostnameEx? Can it be folded in the new option instead, such as a special case PrivateHostname=yes?

Reply:
Hm, `yes` is a valid hostname...
Another option is to make `ProtectHostname=` optionally take a hostname like `ProtectHostname=private:hostname`.